src/applications/caregivers/config/form.js

```diff
@@ -296,37 +346,45 @@ const formConfig = {
       },
     },
     secondaryCaregiversChapter: {
-      title: 'SECONDARY CAREGIVERS',
+      title: 'Secondary Family Caregivers Information',
       pages: {
-        secondaryOneCaregiverIntro: {
+        secondaryCaregiverOneIntro: {
           path: 'secondaryOne-caregiver-intro',
-          title: 'Secondary Caregiver Information',
+          title: ' ',
           uiSchema: {
-            'ui:description': SecondaryCaregiverInfo,
-            [primaryCaregiverFields.hasSecondaryOneCaregiverView]: hasSecondaryOneCaregiverUI,
+            'ui:description': SecondaryCaregiverInfo({ additionalInfo: true }),
+            [primaryCaregiverFields.hassecondaryCaregiverOneView]: hasSecondaryCaregiverOneUI,
           },
           schema: {
             type: 'object',
             properties: {
-              [primaryCaregiverFields.hasSecondaryOneCaregiverView]: {
+              [primaryCaregiverFields.hassecondaryCaregiverOneView]: {
                 type: 'boolean',
               },
             },
           },
         },
-        secondaryOneCaregiver: {
+        secondaryCaregiverOne: {
           path: 'secondaryOne-caregiver-1',
-          title: 'Secondary Caregiver Information',
-          depends: formData => hasSecondaryOneCaregiver(formData),
+          title: 'Secondary Family Caregiver One Information',
```

Comment:
I didn't see `One` in the design; but I did see `(2)`? Is that the index?

Reply:
Cleared it up with @jonathanjnelson correct it